truss/contexts/image_builder/cache_warmer.py

```diff
@@ -1,9 +1,24 @@
-if __name__ == "__main__":
-    import os
-    import sys
+import os
+import sys
+from pathlib import Path
+
+from huggingface_hub import hf_hub_download
+
 
-    from huggingface_hub import hf_hub_download
+def download_file(repo_name, file_name, revision_name=None):
+    secret = None
+    if Path("/etc/secrets/hf_access_token").exists():
+        with open("/etc/secrets/hf_access_token", "r") as secretFile:
```

Comment:
nit: best to save the Path obj in a var and just read_text if it exists: 
https://docs.python.org/3/library/pathlib.html#pathlib.Path.read_text